### PR TITLE
VLLM inference

### PR DESCRIPTION
**Pull request by neubig**
# Description

This PR implements inference for text generation using VLLM, a fast library for text generation.
I still need to benchmark this more extensively, but it seems that this results in a pretty significant speedup in generation time, possibly over 10x!

# References

- [VLLM Library](https://vllm.readthedocs.io/en/latest/)
- Fixes https://github.com/zeno-ml/zeno-build/issues/136

# Blocked by

- NA

### CODE REVIEW

**Comment by cabreraalex** [comment]
And the models are identical? cool!

File: examples/chatbot/config.py [hunk 1]
```diff
@@ -69,12 +69,14 @@
...
                 "model_preset": search_space.Categorical(
                     [
-                        "gpt-3.5-turbo",
+                        # "gpt-3.5-turbo",
                         "gpt2",
                         "gpt2-xl",
                         "llama-7b",
                         "vicuna-7b",
                         "mpt-7b-chat",
+                        "vicuna-7b-v1.3",
```
Comment (cabreraalex) [inline examples/chatbot/config.py:78]: What do you think our norm should be for the default options? e.g. vllm isn't installed by default, should it be in the default config of this example?

**Review by cabreraalex — APPROVED** [review]

File: examples/chatbot/config.py (hunk 1 above)
Comment (neubig) [reply examples/chatbot/config.py:78]: Yeah, that's a good question. vllm is installed through the requirements file for this example now, but there's also an argument that we should make this configurable. I'll try to think about the best way to do that.

Comment (neubig) [reply examples/chatbot/config.py:78]: I will handle this in a future commit: https://github.com/zeno-ml/zeno-build/issues/160

**Pull request merged**